eq-author-api/app.js

```diff
@@ -14,8 +14,12 @@ const repositories = require("./repositories");
 const modifiers = require("./modifiers");
 const schema = require("./schema");
 
+const noir = require("pino-noir");
+
 const app = express();
-const pino = pinoMiddleware();
+const pino = pinoMiddleware({
+  serializers: noir(["req.headers.authorization"], "[Redacted]"),
```

Comment:
This looks unrelated but a good change to stop logging the token. Do you want to do it against master instead?

Reply:
https://github.com/ONSdigital/eq-author-app/pull/184